heron/healthmgr/src/java/com/twitter/heron/healthmgr/HealthManager.java

```diff
@@ -200,12 +192,20 @@ public static void main(String[] args) throws Exception {
         throw new IllegalArgumentException("Invalid mode: " + getOptionValue(cmd, CliArgs.MODE));
     }
 
+    setupLogging(cmd, config);
 
     LOG.info("Static Heron config loaded successfully ");
     LOG.fine(config.toString());
 
+    // load the default config value and override with any command line values
+    String metricSourceClassName = config.getStringValue(PolicyConfigKey.METRIC_SOURCE_TYPE.key());
+    metricSourceClassName = getOptionValue(cmd, CliArgs.METRIC_SOURCE_TYPE, metricSourceClassName);
+
+    String metricsUrl = config.getStringValue(PolicyConfigKey.METRIC_SOURCE_URL.key());
```

Comment:
I realized that the cache manager port is dynamic. As the executor does not know the type of the provider and the metrics cache port does not apply to other metrics providers. So that is not feasible. The metrics cache port is saved in state manager. Health manager has the state manager connection. So the metrics cache provider can read the port from state manager. It will ignore the url provided in the config. If you agree I will create a PR for this?